tests/test_validate_ping.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+# Any copyright is dedicated to the Public Domain.
+# http://creativecommons.org/publicdomain/zero/1.0/
+
+import io
+import json
+
+from glean_parser import validate_ping
+
+
+def test_validate_ping():
+    content = {
```

Comment:
If we're moving the schema off this repo, let's keep this shorter: let's use a small JSON and provide a sample schema stub.